.github/workflows/integration-test.yml

```diff
@@ -15,6 +15,7 @@ jobs:
         uses: actions/checkout@v2
         with:
           repository: 'golemfactory/yagna-integration'
+          branch: 'mwu/multi-payment'
           token: ${{ secrets.YAGNA_WORKFLOW_TOKEN }}
 
       - name: Configure python 3.7
```

Comment:
This workflow is going to be removed in https://github.com/golemfactory/yagna/pull/723. If you'd like to get it running until then anyway, please also update the python version used here to 3.8.

Reply:
Good one, will do when removing the test branch